Makefile

```diff
@@ -52,6 +52,6 @@ checks: test coverage lint
 clean: FORCE
 	find . -name "__pycache__" -exec rm -r {} +
 	find . -name "*.pyc" -exec rm {} +
-	rm -rf .coverage htmlcov
+	rm -rf .coverage.* htmlcov
```

Comment:
I think the same glob pattern should be used in both `.gitignore` and `Makefile` for the sake of consistency. Right now, the two glob patterns differ by a `.`.

If you agree, could you please update this pull request?